packages/headless/src/features/configuration/configuration-slice.ts

```diff
@@ -76,4 +77,7 @@ export const configurationReducer = createReducer(
       .addCase(setOriginLevel3, (state, action) => {
         state.analytics.originLevel3 = action.payload.originLevel3;
       })
+      .addCase(updateActiveTab, (state, action) => {
+        state.analytics.originLevel2 = action.payload;
+      })
```

Comment:
It's a possibility, but there's an action to explicitly set originLevel2 "manually".

That would be very rare exception, though.

Reply:
It's a good point, I baked in the behavior because Coveo expects the tab id and originLevel2 to be in sync.

If someone does not want this, then I think they should not be using tabs?
One could dispatch actions to set the `cq` and `originLevel2` independently.
Another option would be to use the upcoming `StaticFilter` controller, which would only manage a group of expressions, without affecting originLevel2.